### PR TITLE
Added a local player animation fix

### PR DESCRIPTION
**Pull request by Chrisdies**
didn't want to give my own to an open source project so I ported over one from unknowncheats (https://www.unknowncheats.me/forum/counterstrike-global-offensive/348353-accurate-local-player.html)

I tried to not screw with the source that much but adding the detour hook is basically required as messing with the animation netvar fucks up csgosimple 4 some reason. enjoy lol

### CODE REVIEW

File: CSGOSimple/valve_sdk/csgostructs.hpp [hunk 1]
```diff
@@ -249,7 +249,6 @@ class C_BasePlayer : public C_BaseEntity
 		return *(std::array<float, 24>*)((uintptr_t)this + _m_flPoseParameter);
 	}
 
-
```
Comment (Chrisdies) [inline CSGOSimple/valve_sdk/csgostructs.hpp]: guess I forgot I removed a line here

**Comment by mr-nv** [comment]
pretty sure that ms detours are vac detected

**Comment by Chrisdies** [comment]
There is a vac module that detects detours, but it only detects detours that use virtual protect. And that module hasn't been active for multiple years. The ms detours lib doesn't use virtual protect so even if the module was enabled it wouldn't flag you.

**Comment by mr-nv** [comment]
lmao what is wrong with valve

**Comment by zShikdz** [comment]
You don't need detours to make this work, just get creative and this is a base lol? Why does it need an animfix?

**Comment by Chrisdies** [comment]
Go try it out without detours. Good luck. Detouring / hooking update animation is the proper way to prevent the client from updating animations as the netvar does a lot more than just that. This was made more as an example for those using csgo simple, since a lot of people are using legit anti aim or are making it into a rage cheat.

**Comment by mr-nv** [comment]
@Chrisdies he probably meant that you can hook updateclientsideanimation using vmt, look into deadcell's source for that
but yeah, I also think that animation fix is a bit overkill for a base like this

**Comment by zShikdz** [comment]
> Go try it out without detours. Good luck. Detouring / hooking update animation is the proper way to prevent the client from updating animations as the netvar does a lot more than just that. This was made more as an example for those using csgo simple, since a lot of people are using legit anti aim or are making it into a rage cheat.

Im not gonna try because i already did. No need to use detours just use your brain :)

If you want a good way but not as proper as deadcell's just ask for it ill give it to you.

**Comment by Chrisdies** [comment]
Doing some hippy dippy meme with vmt is not what I'm after. It takes me 2.5ms to update the sig for my detour. Try again sweaty.

**Comment by zShikdz** [comment]
> Doing some hippy dippy meme with vmt is not what I'm after. It takes me 2.5ms to update the sig for my detour. Try again sweaty.

'Try again sweaty.'

This shows how imature you are. Im not going to continue this conversation, but I don't recommend using detours in your cheat.

**Comment by lakstick** [comment]
:< Dont work when i try to compile i see this error 
Dangerous module for SAFESH

**Comment by anarh1st47** [comment]
I dont see reason why base need animfix, especially like this. @Vladislavik36 ?

**Comment by anarh1st47** [comment]
This is not needed in base, closing

**Pull request closed without merging**